app/src/main/java/com/twilio/conversations/app/common/extensions/TwilioExtensions.kt

```diff
@@ -207,13 +209,22 @@ suspend fun Conversation.getMessagesBefore(index: Long, count: Int): List<Messag
     })
 }
 
-suspend fun Conversation.sendMessage(message: Message.Options): Message = suspendCoroutine { continuation ->
-    sendMessage(message, object : CallbackListener<Message> {
-
-        override fun onSuccess(message: Message) = continuation.resume(message)
+suspend fun Conversation.sendTextMessage(attributes: Attributes, body: String?): Message {
```

Comment:
If you are switching to SDK `3.0.1` - drop this file completely and use extensions from SDK. In `3.0.1` we already have `Conversation.sendMessage` for this. Like
val message = conversation.senMessage {
    body = text
    attributes = Attributes()
}
Or stay on `3.0.0` for now and create a separate task for future moving to `3.0.1`,

Reply:
Let me drop it in the next PR.
This PR uses just one extension from 3.0.1.

Reply:
Lets copy-paste that one extension into `TwilioExtensions.kt` and stay on `3.0.0` until next PR. It's weird to have almost all extensions from SDK duplicated in demoapp

Reply:
@rusmonster does it fail the compilation? If not, then it should be fine to keep until next PR.

Reply:
@berkus 
It doesn't, but it makes demoapp code ugly which is exactly opposite to what we want to have in the demoapp.
Also taking into account that our priorities could change on any standup - next PR could happen next year....

So my idea is to keep the `main` good now and to improve it in the next PR....

Reply:
We can't drop `TwilioExtensions` totally since some extensions are missing in SDK.

Reply:
yep, good catch! will add them into next sdk release